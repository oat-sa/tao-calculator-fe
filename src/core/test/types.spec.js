/**
 * This program is free software; you can redistribute it and/or
 * modify it under the terms of the GNU General Public License
 * as published by the Free Software Foundation; under version 2
 * of the License (non-upgradable).
 *
 * This program is distributed in the hope that it will be useful,
 * but WITHOUT ANY WARRANTY; without even the implied warranty of
 * MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
 * GNU General Public License for more details.
 *
 * You should have received a copy of the GNU General Public License
 * along with this program; if not, write to the Free Software
 * Foundation, Inc., 51 Franklin Street, Fifth Floor, Boston, MA  02110-1301, USA.
 *
 * Copyright (c) 2018-2023 (original work) Open Assessment Technologies SA ;
 */

import types from '../types.js';

describe('types', () => {
    it('is a namespace', () => {
        expect(types).toEqual(expect.any(Object));
    });

    it('defines types', () => {
        expect(Object.keys(types).length).toBeGreaterThan(0);
        expect(types).toMatchSnapshot();
    });
});
