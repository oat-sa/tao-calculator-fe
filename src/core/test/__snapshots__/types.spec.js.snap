// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`types defines types 1`] = `
{
  "aggregator": "aggregator",
  "constant": "constant",
  "digit": "digit",
  "error": "error",
  "function": "function",
  "operator": "operator",
  "separator": "separator",
  "term": "term",
  "variable": "variable",
}
`;
